zero/src/rpc/jerigon.rs

```diff
@@ -33,16 +53,64 @@ where
             "debug_traceBlockByNumber".into(),
             (target_block_id, json!({"tracer": "zeroTracer"})),
         )
-        .await?;
+        .await?
+        .into_iter()
+        .map(|ztr| ztr.result)
+        .collect::<Vec<_>>();
 
     // Grab block witness info (packed as combined trie pre-images)
-
     let block_witness = cached_provider
         .get_provider()
         .await?
         .raw_request::<_, String>("eth_getWitness".into(), vec![target_block_id])
         .await?;
 
+    let block = cached_provider
+        .get_block(target_block_id, BlockTransactionsKind::Full)
+        .await?;
+
+    let jdts: Vec<Option<JumpDestTableWitness>> = match jumpdest_src {
+        JumpdestSrc::Simulation => vec![None; tx_results.len()],
+        JumpdestSrc::Zero => {
+            process_transactions(
+                &block,
+                cached_provider.get_provider().await?.deref(),
+                tx_results.iter().map(|TxnInfo { traces, meta: _ }| traces), // &tx_traces,
+            )
+            .await?
+        }
+        JumpdestSrc::Jerigon => todo!(),
```

Comment:
```suggestion
        JumpdestSrc::Jerigon => todo!("hybrid server bulk struct log retrieval/local jumpdest table generation not yet implemented"),
```

Reply:
Ok, using it.

Reply:
@cffls do you have an ETA for this hybrid solution? (to know if we can wait for it or if we should ship as is)

also as a side-note, for @atanmarko @einar-polygon, even if the current solution incurs a _lot_ of extra RPC requests for jerigon, it is still better than simulation so apart from a few specific blocks spammed with contract creations, one would _always_ prefer JDT prefetching to the simulation (hence not sure about the utility of having this, rather than a simple todo for updating the logic once the structlog bulk is available).